cms/lib/xblock/authoring_mixin.py

```diff
@@ -51,3 +56,111 @@ def visibility_view(self, _context=None):
         scope=Scope.settings,
         enforce_type=True,
     )
+
+    def editor_saved(self, user, old_metadata, old_content) -> None:  # pylint: disable=unused-argument
+        """
+        Called right *before* the block is written to the DB. Can be used, e.g., to modify fields before saving.
+
+        By default, is a no-op. Can be overriden in subclasses.
+        """
+
+    def post_editor_saved(self, user, old_metadata, old_content) -> None:  # pylint: disable=unused-argument
+        """
+        Called right *after* the block is written to the DB. Can be used, e.g., to spin up followup tasks.
+
+        By default, is a no-op. Can be overriden in subclasses.
+        """
```

Comment:
It seems that we don't need [these attr checks](https://github.com/openedx/edx-platform/blob/619958821dc2b9c965aa6170123f52520ced63be/cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py#L304-L310) anymore.

Reply:
that's right. removed